### DISCUSSION TITLE
Aggiornamento 2022.5.3

### DISCUSSION THREAD

**Discussion by Ilmessaggeromascherato** [indomus/forum#1689] [HUB: Home Assistant]
Ho fatto l'aggiornamento e ora nel log mi compare questo errore: 
Non Capisco nel file config.yaml c'è la riga 

` deafult_config: `

Cosa devo fare ? 
Grazie 

```
Logger: homeassistant.components.homeassistant
Source: components/homeassistant/__init__.py:162 
Integration: Home Assistant Core Integration (documentation, issues) 
First occurred: 11:05:59 (2 occurrences) 
Last logged: 11:07:25

The system cannot restart because the configuration is not valid: Integration error: default_config - Requirements for dhcp not found: ['aiodiscover==1.4.11'].

```

**Comment by snakuzzo** [comment]
Hai installazione in venv ? Nel caso, che versione di Python stai usando ?

**Comment by Ilmessaggeromascherato** [comment]
Ho l’installazione su Rapsberry pi4 Core, questa sera ti dico la versione di Python

**Reply by snakuzzo** [reply]
per favore, non rispondere dalle mail che poi non si capisce niente...e fai reply sotto l'ultimo messaggio.

Quanto a Python, dovresti allinearti ed usare la 3.9.
Io ti consiglio di passare da installazione in venv a docker...per migrare ci metti al massimo 10 minuti e non avrai più problemi

**Reply by Ilmessaggeromascherato** [reply]
Ciao ecco 
la versione Python è la 3.9.9

**Reply by Scialla** [reply]
Hai Core in Venv?

Guarda la risposta di bdraco:
https://github.com/home-assistant/core/issues/61775#issuecomment-998141959
